snmpwalk/metadata.csv

```diff
@@ -1 +1,2 @@
 metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+filebeat.registry.unprocessed_bytes,gauge,"",byte,"",Number of bytes not processed by filebeat yet.,0,filebeat,""
```

Comment:
Hi @wk8 you somehow added the metric to the `snmpwalk` metadata instead of the `filebeat`. Could you please update this?